### ISSUE TITLE
A question about nuscences test set result

### ISSUE THREAD

**Issue by ged0606** [YoushaaMurhij/FMFNet#10]
Hello, thanks for your work.
I have a question about nuscenes test set result listed in this [paper](https://drive.google.com/file/d/1MSU6nLkqAqd2qZDnb0P8tkKLOxieJcyl/view)
![table-1](https://user-images.githubusercontent.com/21331521/170007019-83f21153-9c64-4cd8-9c69-674faa1f7c27.png)

The mAP of FMF-Voxelnet-Base seems wrong, I get 64.6 according per-class AP.

**Comment by YoushaaMurhij** [comment]
We have updated the baseline code. There is a little improvement in the metrics (at least on Waymo). How many sweeps do you use?

**Comment by ged0606** [comment]
Thanks for your reply. Sorry that my expression is not clear.  I have not implement the code, just read the paper list above.

I compute mAP by per class AP listed in the paper, just like: 
 (89.8 + 59.3 + 67.0 + 68.6 + 32.8 + 85.7 + 58.6 + 30.2 + 77.4 + 77.0) / 10 = 64.64.

**Comment by YoushaaMurhij** [comment]
I am using Nuscenes-devkit to calculate the metrics. I will check if weighted average is implemented. Otherwise, I will updated the metrics in a new arxiv version.
Thanks for highlighting this issue.

**Comment by YoushaaMurhij** [comment]
Correct. Thanks for mentioning this mistake. There is a typo in the table.
